docs/book/v6/extended-features/core-and-app.md

```diff
@@ -0,0 +1,34 @@
+# Core and App code structure
+
+In the 6.0 version, the project is split into two main parts: **App** and **Core**.
+
+The purpose is to reach a headless CMS format for easier scalability.
+Headless CMS is a back-end-only content management system that acts primarily as a content repository.
```

Comment:
for consistency, it might be best to simply use 'backend' and 'frontend' everywhere (no dashes, spaces)